planner/core/logical_plans.go

```diff
@@ -617,3 +618,18 @@ type LogicalLock struct {
 
 	Lock ast.SelectLockType
 }
+
+// LogicalWindow represents a logical window function plan.
+type LogicalWindow struct {
+	logicalSchemaProducer
+
+	WindowFuncDesc *aggregation.WindowFuncDesc
```

Comment:
There can be more than one window function in the window operator.
```suggestion
	WindowFuncDesc []*aggregation.WindowFuncDesc
```

Reply:
But currently, we will only have one window function in one window operator. Multiply functions will be done in the future.

Reply:
If it's not hard to support multiply functions in a window operator in the planner phase, I think we should do it this PR.

Reply:
It is not hard, but this pr is already large enough.

Reply:
How about to split this PR into small PRs:
- one PR to implement `WindowFuncDesc` and UT
- one PR to implement logical window operator and convert AST to it, and UT
- one PR to implement physical window operator and transform logical window operator to it, and UT if possible.
- one PR to support `explain` window operator, and UT.

Reply:
I don't think it helps too much, this pr is mainly dealing with the logical window.